GOVERNANCE.md

```diff
@@ -16,64 +17,125 @@ Users activities include (but are not limited to):
 
-### Open Development Process
+#### How to be part of the Advisory Committee 
+* Must be an Organization Member 
+* Elected by the Organization Member on the developer mailing list.
+//TO DO this is new - need to validate. 
 
-OpenRefine development is based on user consensus and open discussion between users. Decision making must be done in a transparent, open fashion (ie. using discussion list and issue list). No decisions about the project’s direction, bug fixes or features may be done in private without community involvement and participation. Discussions must begin at the earliest possible point on a topic; the community’s participation is vital during the entire decision-making process.
+#### Current List of Advisory Committee Members
+* Martin Magdinier
+* Thad Guidry 
+* Antonin Delpeuch. 
+
+## Code For Science and Society 
+
+Since January 2020, OpenRefine is a member project of the Code For Science and Society (CS&S). 
+
+CS&S provides adminstrative and fiscal infrastructure to received and manage funds under CS&S Fiscal Sponsorship Agreement. CS&S is a USA based organization qualified as exempt from federal income tax under Section 501(c)(3) of the Internal Revenue Code (IRC) and classified as a public charity under IRC Sections 509(a)(1) and 170(b)(1)(A)(vi). 
+
+### Manage Funds 
+
+While the vast majority of OpenRefine contributors are volunteers, we recognized that need to attract and retain contributors to either
+- perform time critical maintenance tasks for the project (for example release management, security update, pull request review and comments)
+- address issue that will not be naturally done by the communittee (for example documentation or on long terms project like OpenRefine 4.x)
+
+CS&S currently manage the current funding sources: 
+- **Grants**: Funds are allocated based on the grants requierements. Grants help to secure resources to achieve long term goal or supports software maintenance, growth, development, and community engagement.
+- **Google Summer of Code**: Google Summer of Code mentors are eligible to a USD 500 compensation for their work. 
+- **Donations made through Github sponsors or done directly to CS&S on behalf of OpenRefine**: Usage of the fund is made by the Advisory committee with guidance from Organization Members. 
+
+The CS&S, the Steering and Advisory comittees have experience applying and managing grants. If you are interested in applying to a grant to improve OpenRefine, please share your idea early on the Developer mailing list. It will be a pleasure helping you through your grant application. New grants must be approved by the Advisory Committee. that CS&S retains a 15% handling fees on any contribution to finance their operations (20% for government funding). 
```

Comment:
Fix grammar.
Isn't contribution really to be worded as **donation?**
```suggestion
Combined with CS&S, the Steering and Advisory committees have experience applying and managing grants. If you are interested in applying to a grant to improve OpenRefine, please share your idea early on the Developer mailing list. It will be a pleasure helping you through your grant application. New grants must be approved by the Advisory Committee. that CS&S retains a 15% handling fees on any donation to finance their operations (20% for government funding). 
```